code/game/turfs/closed/minerals.dm

```diff
@@ -24,6 +24,9 @@
 /turf/closed/mineral/Initialize()
 	if (!canSmoothWith)
 		canSmoothWith = list(/turf/closed/mineral, /turf/closed/indestructible)
+	var/ztrait = SSmapping.level_trait(z, ZTRAIT_MINERAL_ICON)
```

Comment:
I can't explain why but the idea of the ztrait changing the icon of the turfs on init doesn't sit right with me.